shared/teams/really-leave-team/index.js

```diff
@@ -1,19 +1,9 @@
 // @flow
 import * as React from 'react'
 import * as Constants from '../../constants/teams'
-import {
-  Avatar,
-  Box,
-  ButtonBar,
-  HeaderOnMobile,
-  Icon,
-  MaybePopup,
-  ProgressIndicator,
-  ScrollView,
-  Text,
-  WaitingButton,
-} from '../../common-adapters'
-import {globalStyles, globalMargins, isMobile} from '../../styles'
+import {Avatar, Box, HeaderOnMobile, Icon, MaybePopup, ProgressIndicator} from '../../common-adapters'
+import ConfirmModal from '../../common-adapters/confirm-modal'
+import {globalStyles, globalMargins} from '../../styles'
```

Comment:
Maybe switch these to `Kb` and `Styles` while you are here?

Reply:
I thought I couldn't just import * from common-adapters because of the loop?